src/main/io/ledstrip.c

```diff
@@ -1006,11 +1022,13 @@ void ledStripInit(ledConfig_t *ledConfigsToUse, hsvColor_t *colorsToUse, failsaf
     ledStripInitialised = false;
 }
 
+#define ROTATION_SEQUENCE_LED_WIDTH 0x03       // 0x03 = 2 leds ON ; 0x05 = 3 leds ON
+
 void ledStripEnable(void)
 {
     reevalulateLedConfig();
     ledStripInitialised = true;
-
+    maskRotationPhase = ROTATION_SEQUENCE_LED_WIDTH | (ROTATION_SEQUENCE_LED_WIDTH << (ledsInRingCount>>1));
```

Comment:
Ok,  I used to work with microchip compilator which prefer call the function divisor rather than just used a shift. (mean when you look to the assembly code). I'm happy Gcc is a bit more clever :D